src/include/pbs_ifl.h

```diff
@@ -338,6 +339,8 @@ extern "C" {
 #define ATTR_NODE_TopologyInfo	"topology_info"
 #define ATTR_NODE_MaintJobs	"maintenance_jobs"
 #define ATTR_NODE_VnodePool	"vnode_pool"
+#define ATTR_NODE_current_eoe   "current_eoe"
+#define ATTR_NODE_power_provisioning   "power_provisioning"
```

Comment:
With this you can keep power provisioning independent of OS/application provisioning. If you have both Power and OS/Application provisioning enabled at server and want only power provisioning on a particular node and only OS provisioning enabled on other nodes, you will need a separate attribute to do it.